docs/commands/rpc-protocol.md

```diff
@@ -457,7 +455,7 @@ Boolean, false by default. Additionally sorts each account's blocks by their amo
 **Optional "include_only_confirmed"**
 
 _version 19.0+_  
-Boolean, false by default. Only returns blocks which have their confirmation height set or are undergoing confirmation height processing.
+Boolean, true by default. Only returns blocks which have their confirmation height set or are undergoing confirmation height processing. If false, unconfirmed blocks will also be returned.
```

Comment:
It is true by default, from v22 onwards only.

Reply:
Updated